collect_app/src/main/res/color/magenta_theme_tint_list.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think this should be more explicitly named so it's obvious it's for the image buttons.